bench/worker/worker_ci_bench.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'base'
+
+class WorkerCIBenchmark < Base
+  def run_benchmark
+    benchmark_reports(10_000)
+
+    res = benchmark_reports(100_000)
+    rss = `ps -o rss #{Process.pid}`.lines.last.to_i
+
+    warn "=" * 70
+    warn Benchmark::CAPTION
+    warn res.format(Benchmark::Tms::FORMAT).chop + " (#{res.label})"
+    warn "=" * 70
+    warn "Memory usage: #{rss} KB"
+    warn "=" * 70
+  end
+end
+
+WorkerCIBenchmark.new.run
```

Comment:
```suggestion
if __FILE__ == $PROGRAM_NAME
  WorkerCIBenchmark.new.run
end
```

Reply:
If I do this the rake task doesn't run the benchmark

Reply:
I don't like it but ok, now I remember not liking it before. Maybe refactor some day.

Reply:
What do you mean? you don't like running it as a rake task?